customer-churn/run_kubeflow_pipeline.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+
+from materializer.customer_materializer import cs_materializer
+from pipelines.data_analysis_pipeline import data_analysis_pipeline
+from pipelines.training_pipelines import training_pipeline
+from steps.data_process import (
+    drop_cols,
+    encode_cat_cols,
+    handle_imbalanced_data,
+)
+from steps.data_splitter import data_splitter
+from steps.evaluation import evaluation
+from steps.ingest_data import ingest_data
+from steps.trainer import model_trainer
+from steps.visualizer import (
+    visualize_statistics,
+    visualize_train_test_statistics,
+)
+from zenml.integrations.mlflow.mlflow_utils import get_tracking_uri
+
+
+def analyze_pipeline():
+    """Pipeline for analyzing data."""
+    analyze = data_analysis_pipeline(
+        ingest_data(),
+        data_splitter(),
+    )
+    analyze.run()
+    visualize_statistics()
+    visualize_train_test_statistics()
+
+
+def training_pipeline_run():
+    """Pipeline for processing data."""
+    train_pipeline = training_pipeline(
+        ingest_data(),
+        encode_cat_cols(),
+        handle_imbalanced_data(),
+        drop_cols(),
+        data_splitter(),
+        model_trainer().with_return_materializers(cs_materializer),
+        evaluation().with_return_materializers(cs_materializer),
```

Comment:
Until now we have always used keyword arguments instead of positional arguments when instantiating the pipeline, personally I feel it would be good to stick with that.

Reply:
done!